<?xml version="1.0" encoding="utf-8" ?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
        "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="" lang="">
<head>
  <title>Test Results Report - Assertions Test</title>
  <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
  <meta name="description" content="TestNG unit test results." />
  <link href="reportng.css" rel="stylesheet" type="text/css" />
    <script type="text/javascript" src="reportng.js"></script>
</head>
<body>
<h1>Assertions Test</h1>
<p>
  Test duration: 0.017s
</p>



  <table class="resultsTable">
    <tr><th colspan="3" class="header failed">Failed Tests</th></tr>
          <tr>
        <td colspan="3" class="group">testcases.TestCase2</td>
      </tr>
            
<tr>
  <td class="method">
                  checkCondition
      </td>
  <td class="duration">
    0.002s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <a href="C:\Screenshot\error.jpg" target="_blank">Screenshot capture</a>
                                <br>
                                <a href="C:\Screenshot\error.jpg" target="_blank"> <img height=200 width=200 src="C:\Screenshot\error.jpg"></a>
                  </div>
    

        <a href="javascript:toggleElement('exception-0', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: element not found did not expect to find [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-0">
              org.testng.Assert.fail(Assert.java:97)<br />
              org.testng.Assert.failNotEquals(Assert.java:969)<br />
              org.testng.Assert.assertTrue(Assert.java:43)<br />
              testcases.TestCase2.checkCondition(TestCase2.java:28)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.base/java.lang.reflect.Method.invoke(Method.java:567)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)<br />
              org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)<br />
              org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)<br />
              org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)<br />
              org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)<br />
              org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)<br />
              java.base/java.util.ArrayList.forEach(ArrayList.java:1507)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:770)<br />
              org.testng.TestRunner.run(TestRunner.java:591)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:402)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:304)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1102)<br />
              org.testng.TestNG.runSuites(TestNG.java:1032)<br />
              org.testng.TestNG.run(TestNG.java:1000)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  forceFailTestcase
      </td>
  <td class="duration">
    0.003s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <a href="C:\Screenshot\error.jpg" target="_blank">Screenshot capture</a>
                                <br>
                                <a href="C:\Screenshot\error.jpg" target="_blank"> <img height=200 width=200 src="C:\Screenshot\error.jpg"></a>
                  </div>
    

        <a href="javascript:toggleElement('exception-1', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: forcing failure</b></a><br />
    <div class="stackTrace" id="exception-1">
              org.testng.Assert.fail(Assert.java:97)<br />
              testcases.TestCase2.forceFailTestcase(TestCase2.java:34)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.base/java.lang.reflect.Method.invoke(Method.java:567)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)<br />
              org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)<br />
              org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)<br />
              org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)<br />
              org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)<br />
              org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)<br />
              java.base/java.util.ArrayList.forEach(ArrayList.java:1507)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:770)<br />
              org.testng.TestRunner.run(TestRunner.java:591)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:402)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:304)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1102)<br />
              org.testng.TestNG.runSuites(TestNG.java:1032)<br />
              org.testng.TestNG.run(TestNG.java:1000)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
<tr>
  <td class="method">
                  validateTitles
      </td>
  <td class="duration">
    0.012s
  </td>
  <td class="result">
        
    
            <div class="testOutput">
                        <a href="C:\Screenshot\error.jpg" target="_blank">Screenshot capture</a>
                                <br>
                                <a href="C:\Screenshot\error.jpg" target="_blank"> <img height=200 width=200 src="C:\Screenshot\error.jpg"></a>
                  </div>
    

        <a href="javascript:toggleElement('exception-2', 'block')" title="Click to expand/collapse"><b>java.lang.AssertionError: The following asserts failed:<br/>
	expected [Gmail.com] but found [gmail.com],<br/>
	textbox not found expected [false] but found [true],<br/>
	images did not match expected [true] but found [false]</b></a><br />
    <div class="stackTrace" id="exception-2">
              org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)<br />
              org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)<br />
              testcases.TestCase2.validateTitles(TestCase2.java:22)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)<br />
              java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)<br />
              java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)<br />
              java.base/java.lang.reflect.Method.invoke(Method.java:567)<br />
              org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)<br />
              org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)<br />
              org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)<br />
              org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)<br />
              org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)<br />
              org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)<br />
              org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)<br />
              org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)<br />
              java.base/java.util.ArrayList.forEach(ArrayList.java:1507)<br />
              org.testng.TestRunner.privateRun(TestRunner.java:770)<br />
              org.testng.TestRunner.run(TestRunner.java:591)<br />
              org.testng.SuiteRunner.runTest(SuiteRunner.java:402)<br />
              org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)<br />
              org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)<br />
              org.testng.SuiteRunner.run(SuiteRunner.java:304)<br />
              org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)<br />
              org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)<br />
              org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)<br />
              org.testng.TestNG.runSuitesLocally(TestNG.java:1102)<br />
              org.testng.TestNG.runSuites(TestNG.java:1032)<br />
              org.testng.TestNG.run(TestNG.java:1000)<br />
              org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)<br />
              org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)<br />
              org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)<br />
                </div>
      </td>
</tr>
      </table>


  
</body>
</html>
